src/Integration.Vsix/SonarLintTagger/TaggerProvider.cs

```diff
@@ -50,23 +49,24 @@ internal sealed class TaggerProvider : IViewTaggerProvider, ITableDataSource, II
         internal readonly ITextDocumentFactoryService TextDocumentFactoryService;
         internal readonly IContentTypeRegistryService ContentTypeRegistryService;
         internal readonly IFileExtensionRegistryService FileExtensionRegistryService;
-        internal readonly _DTE dte;
+        internal readonly IServiceProvider serviceProvider;
 
         private readonly List<SinkManager> managers = new List<SinkManager>();
         private readonly TrackerManager taggers = new TrackerManager();
 
         private readonly ISonarLintDaemon daemon;
         private readonly ISonarLintSettings settings;
-
+        private readonly ISonarLintOutput logger;
 
         [ImportingConstructor]
         internal TaggerProvider(ITableManagerProvider provider,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 8 parameters, which is greater than the 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS107)